File: Pattern33.go
package main

import "fmt"

func main() {
	var n int
	fmt.Printf("Enter the row number:")
	fmt.Scanln(&n)
	var flag int = 1
	for i := 1; i <= 2*n-1; i++ {
		if i == n {
			flag = 2*n - 1
		}
		for k := 1; k <= n-flag; k++ {
			fmt.Printf(" ")
		}
		for j := 1; j <= flag; j++ {
			fmt.Printf("+")
		}
		flag = 1
		fmt.Println()
	}
}

File: Pattern33.py


n =  int(input("Enter the row number: "))
for i in range(1,2*n):
    if i==n:
        
        for j in range(1,2*n-1):
            print("*",end='')
        print("*")
        continue
    for j in range (0,n-1):
        print(" ",end='')
    print("*")
